### ISSUE TITLE
Adding Header to drawer-toolbar section when in drawer item

### ISSUE THREAD

**Issue by ciaranlp** [adaptlearning/adapt_framework#1374]
For example - 

when resources extension is active and the user goes to the drawer, once you click on the `resources-drawer` in the `drawer-toolbar` there is `drawer-back-button` and `drawer-close-button`, it would be great to have the title of the `drawer item`, `resources` in this example.

**Comment by sarveshwar-gavhane** [comment]
@ciaranlp  is it also applicable for pageLevelProgress drawer item ??

**Comment by ciaranlp** [comment]
@sarveshwar-gavhane Yes for consistency it would be a good idea to add it there also.

**Comment by moloko** [comment]
I think it should just be the case that any drawer item that has a title should have that title displayed when that drawer item is being shown

**Comment by barryo-learningpool** [comment]
I've had a go at this one.  Here is the PR:
https://github.com/adaptlearning/adapt_framework/pull/1627
